code/modules/clothing/head/misc_special.dm

```diff
@@ -238,22 +238,31 @@
 	flags_inv = HIDEHAIR
 	var/hair_style = "Very Long Hair"
 	var/hair_color = "#000"
-	var/adjustablecolor = TRUE //can color be changed manually?
+	var/natural = FALSE //monkestation edit
 
+//Monkestation edits: wigs now use a radial menu, there is only one wig that can toggle "natural mode", wigs no longer use overlays for their item icon
 /obj/item/clothing/head/wig/Initialize(mapload)
+	hair_style = pick(GLOB.hair_styles_list) //Bald now just defaults
+	hair_color = "#[random_short_color()]"
 	. = ..()
 	update_icon()
 
 /obj/item/clothing/head/wig/update_icon()
-	cut_overlays()
 	var/datum/sprite_accessory/S = GLOB.hair_styles_list[hair_style]
-	if(!S)
-		icon_state = "pwig"
+	if(!S || S.icon_state == "bald" || S.icon_state == "hair_bald2")
+		icon = 'icons/mob/human_face.dmi'
+		icon_state = "hair_vlong"
 	else
-		var/mutable_appearance/M = mutable_appearance(S.icon,S.icon_state)
-		M.appearance_flags |= RESET_COLOR
-		M.color = hair_color
-		add_overlay(M)
+		icon = S.icon
+		icon_state = S.icon_state
+	color = hair_color
+
+/obj/item/clothing/head/wig/equipped(mob/living/carbon/human/user, slot)
```

Comment:
Natural hair mode appears to default to pure white and never changes.

Reply:
It was actually simpler than that!  Funnily enough I attempted to solve it with regex before realizing how straightforward it actually was.

Byond seems to auto-convert between 3-digit and 6-digit color codes.  The problem is that I was setting the same variable that update_icon updates.  It should work now!